### PR TITLE
update progress assessment page

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
add images , add fields, use short key in  json file 
use view-relevant-information to show "relevant website and docs, other relevant websites"
image

### CODE REVIEW

**Review by tahmeedshah — APPROVED** [review]

**Comment by liyuanmontreal** [comment]
fix an issue , hide legend when no content in national indicators section.
image

**Review by tahmeedshah — APPROVED** [review]

**Review by tahmeedshah — APPROVED** [review]

File: app/views/forms/view/chm/national-assessment.vue [hunk 1]
```diff
@@ -1,123 +1,159 @@
...
+                    <ng v-vue-ng:km-value-ml  :value="document.assessmentText" :locales="locale" html></ng> 
+                </div> 
+            </section>
 
-                    <div v-if="document.documentLinks">                        
-                        <label>{{ t("relevantWebsitesLinksAndFiles") }} </label>
-                       <div class="km-value" compare-val>                 
-                            <ng v-vue-ng:km-link-list v-model:ng-model="document.documentLinks"  ></ng>                    
-                        </div>
+            <section v-if="(document.nationalIndicatorsUsed===false) || (document.nationalIndicatorsUsed && document.nationalIndicators) || document.strategicPlanIndicators || document.implementationActivities">
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/national-assessment.vue:71]: move this big conditions to computed prop

File: app/app-text/views/reports/chm/national-assessment.json [hunk 2]
```diff
@@ -2,19 +2,24 @@
...
-    "summaryOfTheAssessmentOfProgressesTowardTheImplementationOfTheSelectedTarget":"Summary of the assessment of progresses toward the implementation of the selected target",
+    "category":  "Category of progress towards the implementation of the selected target",
+    "rate":  "Rate of progresses toward the implementation of the selected target",
+    "date":  "Date the assessment was done",
+    "summary":"Summary of the assessment of progresses toward the implementation of the selected target",
     "indicatorsAndActivities": "Indicators and Activities",
-    "indicatorsUsedInThisAssessment":"Indicator(s)used in this assessment",    
-    "anyOtherToolsOrMeansUsedForAssessingProgress":"Any other tools or means used for assessing progress",
-    "relevantWebsitesLinksAndFiles":"Relevant websites, links, and files",
+    "indicators":"Indicator(s)used in this assessment",
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/app-text/views/reports/chm/national-assessment.json]: fixed

File: app/views/forms/view/chm/national-assessment.vue [hunk 3]
```diff
@@ -1,123 +1,159 @@
...
+                    <label>{{ t("linkToActivity") }}</label>
+                    <div class="km-value">
+                        <div v-for="item in document.implementationActivities" >
+                            {{lstring(item.title ,locale) }}
+                        </div>
                     </div>
+                </div>                
+            </section>
+
+            <section v-if="document.confidence || document.confidenceInfo || document.adequacy||document.adequacyDescription" >
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/national-assessment.vue:104]: computed will be good here

File: app/views/forms/view/chm/national-assessment.vue [hunk 4]
```diff
@@ -1,123 +1,159 @@
...
+
+            <section v-if="document.documentText || document.documentLinks">
+                <legend>{{ t("relevantDocumentAndInformation") }}</legend>
+                <view-relevant-information :relevant-information="document.documentText" :relevant-documents="document.documentLinks" :locale="locale">                 
+                    <template v-slot:information>
+                        <label>{{ t("relevantInfo") }} </label>   
+                    </template>
+                    <template v-slot:document>
+                        <label>{{ t("relevantWebsites") }} </label> 
+                    </template>
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/national-assessment.vue:140]: does it not have a default label, which is common most of the time?

File: app/views/forms/view/chm/national-assessment.vue (hunk 1 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/national-assessment.vue:71]: fixed

File: app/views/forms/view/chm/national-assessment.vue [hunk 5]
```diff
@@ -1,123 +1,159 @@
...
-                    </div>
 
-                    <div v-if="document.confidenceInfo">
-                        <label>{{ t("levelOfConfidenceOfTheAboveAssessment") }} </label>   
-                        <ng v-vue-ng:km-value-ml  :value="document.confidenceInfo" :locales="locale" html ></ng>  
+                <div v-if="document.strategicPlanIndicators">
+                    <label>{{ t("linkToIndicator") }}</label>
+                    <div class="km-value">
+                        <div v-for="item in strategicPlanIndicators" >
+                            {{lstring(item.title ,locale) }}
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/views/forms/view/chm/national-assessment.vue:89]: fixed

File: app/views/forms/view/chm/national-assessment.vue (hunk 3 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/national-assessment.vue:104]: fixed

**Comment by liyuanmontreal** [comment]
image
image

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**